app/Http/Controllers/Common/SettingsController.php

```diff
@@ -384,15 +384,7 @@ public function getMails()
                  return "<input type='checkbox' class='email' value=".$model->id.' name=select[] id=check>';
              })
                            ->addColumn('date', function ($model) {
-                               $date = $model->date;
-                               if ($date) {
-                                   $date1 = new \DateTime($date);
-                                   $tz = \Auth::user()->timezone()->first()->name;
-                                   $date1->setTimezone(new \DateTimeZone($tz));
-                                   $finalDate = $date1->format('M j, Y, g:i a ');
-                               }
-
-                               return $finalDate;
+                               return getDateHtml($model->date);
```

Comment:
why controller changes are part of theme update PR?

Reply:
Reverted. Will make this part of other PR.